### PR TITLE
Asim/new network package

### PR DESCRIPTION
**Pull request by oshezaf**
Required items, please complete
   
   Change(s):
   - New build for ASim network package

### CODE REVIEW

File: Parsers/ASimNetworkSession/ARM/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Advanced Security Information Model (ASIM) NetworkSession parsers
```
Comment (yaronMSFT) [inline Parsers/ASimNetworkSession/ARM/README.md:1]: I think this readme needs to be under Parsers/ASimNetworkSession folder

File: Parsers/ASimNetworkSession/ARM/vimNetworkSessionPaloAltoCEF/vimNetworkSessionPaloAltoCEF.json [hunk 2]
```diff
@@ -2,33 +2,40 @@
...
...trcat_array( array_slice(hostelements,1,-1), '.')\n  | extend DvcFQDN = iff(Dvc contains \".\",Dvc,\"\" )\n        , DvcDomainType=iff(Dvc contains \".\",\"FQDN\",\"\" )\n| project-away hostelements\n| lookup Actions on DeviceAction\n// Action post filtering\n| where (array_length(dvcaction)==0 or DvcAction has_any (dvcaction))\n| project-rename\n    DstMacAddr=DestinationMACAddress\n    , SrcMacAddr=SourceMACAddress\n    , DstIpAddr=DestinationIP\n    , DstPortNumber=DestinationPort\n    , DstNatPortNumber=DestinationTranslatedPort\n  , SrcPortNumber=SourcePort\n    , SrcIpAddr=SourceIP\n  , SrcNatPortNumber=SourceTranslatedPort\n  , DvcOutboundInterface=DeviceOutboundInterface\n  , DvcInboundInterface=DeviceInboundInterface\n  , EventMessage=Message\n  , DvcOriginalAction=DeviceAction\n// -- Aliases\n| extend\n  IpAddr = SrcIpAddr,\n  Rule=NetworkRuleName,\n  Dst=DstIpAddr,\n  // Host=DstHostname,\n  User=DstUsername,\n  Duration=NetworkDuration,\n  SessionId=NetworkSessionId,\n  EventEndTime =EventStartTime,\n  Src=SrcIpAddr\n};\nNWParser(starttime, endtime, srcipaddr_has_any_prefix, dstipaddr_has_any_prefix, dstportnumber, hostname_has_any, dvcaction, eventresult, disabled)\n",
```
Comment (yaronMSFT) [inline Parsers/ASimNetworkSession/ARM/vimNetworkSessionPaloAltoCEF/vimNetworkSessionPaloAltoCEF.json:38]: did we change the parser?

**Review by yaronMSFT — CHANGES_REQUESTED** [review]

File: Parsers/ASimNetworkSession/ARM/README.md (hunk 1 above)
Comment (oshezaf) [reply Parsers/ASimNetworkSession/ARM/README.md:1]: Thanks. Copied.

File: Parsers/ASimNetworkSession/ARM/vimNetworkSessionPaloAltoCEF/vimNetworkSessionPaloAltoCEF.json (hunk 2 above)
Comment (oshezaf) [reply Parsers/ASimNetworkSession/ARM/vimNetworkSessionPaloAltoCEF/vimNetworkSessionPaloAltoCEF.json:38]: I checked the history, and we did. In general, the repackaging is there to ensure all YAML updates are in.

**Pull request merged**